WordPress/src/main/AndroidManifest.xml

```diff
@@ -81,6 +75,27 @@
                 <!-- Samsung multiwindow support -->
                 <category android:name="android.intent.category.MULTIWINDOW_LAUNCHER" />
             </intent-filter>
+
+            <intent-filter>
```

Comment:
Yes. The reason for this is that DeepLinkingIntentReceiverActivity opens up only that Activity, meaning when a user goes back, it closes out the app instead of going to the Base `WPMainActivity`.

This way, when we open through `WPLaunchActivity`, we verify that not only is the user able to go through the back stack, but the Activity Stack is started correctly where there won't be any duplicates.

Reply:
Just to make sure we are on the same page -> We can use `DeepLinkingIntentReceiverActivity` along with the `TaskStackBuilder` right?